todo/management/commands/hopper.py

```diff
@@ -56,37 +56,47 @@ def handle(self, *args, **options):
 
         fake = Faker()  # Use to create user's names
 
+        user_model = get_user_model()
+        group_field = getattr(user_model, setting("TODO_USER_GROUP_ATTRIBUTE"), "groups")
+        # Django's ManyToManyRel is a little odd in that its directional sense 
+        # is not guranteeed and must be tested for. One of these models is User, 
+        # the other is the Groups model (the one group_field points to)  
+        candidates = (group_field.rel.model, group_field.rel.related_model)
```

Comment:
Wait, didn't we just do this in another function? This is not DRY. Find a way to make this (or whatever we decide to go with) reusable (and thus testable).

Reply:
True, this can go to a small utility function.